karma.conf.saucelabs.js

```diff
@@ -24,6 +24,14 @@ var browsers = {
     browserName: 'MicrosoftEdge',
     platform: 'Windows 10',
     version: '13.10586'
+  },
+  sl_android: {
```

Comment:
Very old but still used by a lot of people. I added it for the legacy version. With that argument, you could also kill IE11, also very old but still, we should support it for enterprise/corporate customers